src/core/viz/expressions/color/named-color.js

```diff
@@ -36,6 +36,9 @@ export class NamedColor extends BaseExpression {
         this.name = colorName;
         this.color = this._nameToRGB(this.name);
     }
+    get value() {
```

Comment:
I think we can do that, but in other PR because there are some expressions that might not have a working the getter value so it should be done carefully.

Reply:
I'm ok with doing it later, but, are you sure that those expressions exist?

Reply:
Maybe but it is something we need to check